docs/application/dotnet/samples/ui/overview.md

```diff
@@ -342,7 +346,7 @@ for TD:
 			<p><img alt="" height="225" src="media/tv5textsample.png" width="400" /></p>
 			</td>
 			<td>
-            <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/TextSample" target="_blank"><strong>(T) Text Sample (NUI)</strong></a></p>
+            <p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/NUI/TextSample" target="_blank"><strong>(T) Text Sample (NUI)</strong></a></p>
```

Comment:
The parts of the image are blur ( for e.g text), can you please reproduce a better quality image.

Reply:
This PR is for quick fix for broken link. Better images can be updated after getting newer images from each sample app developers. It would need 1 or 2 weeks by super slow feedback.